Apps/g.Tec/g.NEEDaccess/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+cmake_minimum_required(VERSION 3.0)
+project(gNEEDaccess VERSION 0.0.1)
+
+# load LSLAppBoilerplate if not done already
+if(NOT TARGET LSL::lsl)
+    if(NOT LSL_ROOT)
+        set(LSL_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../../build/lsl_Release/lslinstall/LSL")
+        get_filename_component(LSL_ROOT ${LSL_ROOT} ABSOLUTE)
+        if(NOT EXISTS ${LSL_ROOT})
+            message(FATAL_ERROR "When building a single app out of tree you need to set LSL_ROOT ( cmake -DLSL_ROOT=/path/to/installed/lsl)")
+        endif()
+    endif()
+    list(APPEND CMAKE_MODULE_PATH "${LSL_ROOT}/cmake")
+    include(LSLAppBoilerplate)
+endif()
+
+# GENERAL CONFIG #
+
+set(CMAKE_INSTALL_PREFIX ${LSL_ROOT}/..)
+
+LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+# set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
+
+# THIRD PARTY LIBRARIES #
+
+# Qt
+if(NOT Qt5_DIR)
+    set(Qt5_DIR "C:\\Qt\\5.9.1\\msvc2015_64\\lib\\cmake\\Qt5")
+endif()
+find_package(Qt5 REQUIRED COMPONENTS Core Xml Gui Widgets)
+
+# Emotiv SDK
+find_package(gNEEDaccessSDK)
+
+
+# APPLICATION #
+
+SET(gNEEDaccess_SRC)
+SET(gNEEDaccess_INCL_DIRS)
+SET(gNEEDaccess_REQ_LIBS)
+
+# Sources
+LIST(APPEND gNEEDaccess_SRC
```

Comment:
I don't know that part of CMake, but FindPackage with an imported target could set include directories and linker targets automatically